README.md

```diff
@@ -1,142 +1,10 @@
-# Supported Windows Server 2016 Version 1709 (Fall Creators Update) amd64 tags
+dotnet-framework-docker
```

Comment:
This isn't really a real README. The README should describe the repo and the project, not just be a meta-DH README. I'll add some different text.